packages/scm/src/browser/scm-preferences.ts

```diff
@@ -0,0 +1,61 @@
+/********************************************************************************
+ * Copyright (C) 2020 Ericsson and others.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License v. 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0.
+ *
+ * This Source Code may also be made available under the following Secondary
+ * Licenses when the conditions for such availability set forth in the Eclipse
+ * Public License v. 2.0 are satisfied: GNU General Public License, version 2
+ * with the GNU Classpath Exception which is available at
+ * https://www.gnu.org/software/classpath/license.html.
+ *
+ * SPDX-License-Identifier: EPL-2.0 OR GPL-2.0 WITH Classpath-exception-2.0
+ ********************************************************************************/
+
+import { interfaces } from 'inversify';
+import {
+    createPreferenceProxy,
+    PreferenceProxy,
+    PreferenceService,
+    PreferenceSchema,
+    PreferenceContribution
+} from '@theia/core/lib/browser/preferences';
+
+export const scmPreferenceSchema: PreferenceSchema = {
+    type: 'object',
+    properties: {
+        'scm.defaultViewMode': {
+            type: 'string',
+            enum: ['tree', 'list'],
+            enumDescriptions: [
+                'Show the repository changes as a tree.',
+                'Show the repository changes as a list.'
+            ],
+            description: 'Controls the default source control view mode.',
+            default: 'list'
+        }
+    }
+};
+
+export interface ScmConfiguration {
+    'scm.defaultViewMode': 'tree' | 'list'
+}
```

Comment:
The tree view uses 'tree' | 'flat'.  I don't care which but one or the other should be changed for consistency.

Reply:
@westbury we generally want to align with the vscode preference values so we are able to support settings under `.vscode/settings.json`. Would you like me to refactor 'flat' to 'list' in the codebase?

Reply:
Yes, and the hover text from 'Toggle to Flat View' to 'Toggle to List View'.